src/app/components/CarouselCoins.tsx

```diff
@@ -44,7 +48,10 @@ export default function CarouselCoins() {
             key={coin.id}
             className="sm:basis-1/2 md:basis-1/3 lg:basis-1/4 xl:basis-1/5"
           >
-            <div className="h-[88px] dark:bg-[#191925] bg-[#FFFFFF] flex items-center rounded-lg cursor-pointer">
+            <div
+              onClick={() => handleCarousel(coin.id)}
```

Comment:
We should not add onClick on a div, it is semantically not correct. You should replace this div with a button and remove the styling if you don't want styling.